src/d3a/setup/sam_config.py

```diff
@@ -38,8 +38,7 @@ def get_setup(config):
                          appliance=PVAppliance()),
                     Area('H1 Storage1',
                          strategy=StorageStrategy(battery_capacity_kWh=12 * 1.2,
-                                                  max_abs_battery_power_kW=12 * 1.2,
-                                                  initial_capacity_kWh=0.6 * 12 * 1.2),
+                                                  max_abs_battery_power_kW=12 * 1.2),
```

Comment:
Please incorporate this comment [here](https://github.com/gridsingularity/d3a/pull/519#discussion_r329089443) in rest of PR too. Thanks!

Reply:
ok, will do!